### PR TITLE
Order/create

### PR DESCRIPTION
**Pull request by jidboris**
functionalities for Users to make delivery order

### CODE REVIEW

**Comment by jidboris** [comment]
PR as requested

File: src/V1/APIs/Controllers/Order/orderController.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import express, { Request, Response, NextFunction } from 'express';
+import OrderService from '../../Services/Orders/orderService';
+import { Orders, OrderType } from '../../Models/orders';
+const newOrder = new OrderService();
+
+export default class orderController {
+   public async createOrder(req: Request, res: Response, next: NextFunction) {
+   return await newOrder.createOrders(req, res, next)
```
Comment (forestbaba) [inline src/V1/APIs/Controllers/Order/orderController.ts]: Await is redundant here...Remove it

File: src/V1/APIs/Controllers/Order/orderController.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+};
+   public async fetchAllOrder(req: Request, res: Response, next: NextFunction)  {
+   return await newOrder.getAllOrder (req, res, next);
```
Comment (forestbaba) [inline src/V1/APIs/Controllers/Order/orderController.ts]: Same here

Comment (forestbaba) [inline src/V1/APIs/Controllers/Order/orderController.ts]: Same here

File: src/V1/APIs/Controllers/Order/orderController.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+};
+   public async fetchOrderById(req: Request, res: Response, next: NextFunction) {
+      return await newOrder.getOrderById(req, res, next);
```
Comment (forestbaba) [inline src/V1/APIs/Controllers/Order/orderController.ts]: Same here

**Review by forestbaba — APPROVED** [review]

**Pull request merged**